web/src/components/Header.js

```diff
@@ -6,37 +6,111 @@ import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { push } from 'connected-react-router';
 
-
+import { getIsAdmin } from '../reducers/user';
 import { t } from '../i18n';
 
+import Icon, { faChevronDown, faChevronLeft, faEdit, faSignOutAlt } from './Icons';
+
 class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ariaExpanded: props.ariaExpanded,
+    };
+    this.node = React.createRef();
+  }
+
+  handleOutsideClick = e => {
+    if (this.node.current.contains(e.target)) {
+      return;
+    }
+    this.setState({ ariaExpanded: false })
+    // remove the global click handler when the dropdown is collapsed
+    document.removeEventListener('click', this.handleOutsideClick);
+  };
+
   handleLogout = e => {
     e.preventDefault();
     const { pushRoute } = this.props;
+    this.toggleDropdown();
     pushRoute('/logout');
   };
 
+  toggleDropdown = () => {
+    this.setState(prev => {
+      if (!prev.ariaExpanded) {
+        // add global click handler when the dropdown is expanded
+        document.addEventListener('click', this.handleOutsideClick);
+       } else {
+        // remove the global click handler when the dropdown is collapsed
+        document.removeEventListener('click', this.handleOutsideClick);
+      }
+      return { ariaExpanded: !prev.ariaExpanded };
+    });
+  };
+
   render() {
-    const { authenticated } = this.props;
+    const { authenticated, currentUser, isAdmin, showSiteTitle } = this.props;
+    const { ariaExpanded } = this.state;
     return (
-      <header className="header__slim">
+      <header ref={this.node}>
         <div className="ds-l-container">
           <div className="ds-l-row">
-            <div className="ds-l-col--12 ds-l-md-col--4">
-              <Link to="/" className="site-title">
-                {t('titleBasic')}
-              </Link>
+            <div className="ds-l-col--12 ds-l-md-col--4 site-title">
+              {showSiteTitle &&
+                <Link to="/">
+                  {t('titleBasic')}
+                </Link>
+              }
+              {!showSiteTitle && !authenticated &&
+                <Link to="/">
+                  {t('titleBasic')}
+                </Link>
+              }
+              {!showSiteTitle && authenticated && (
+                <Link to="/">
+                  <Icon icon={faChevronLeft} size="sm" />
+                  { isAdmin ? "Admin dashboard" : `${currentUser.state.id.toUpperCase()} APD home` }
```

Comment:
Small thing, but can we title case this to "APD Home" vs "APD home" to match the design sketch?